README.md

```diff
@@ -36,71 +36,68 @@ using System.Diagnostics;
 using System.Linq;
 using System.Threading.Tasks;
 
-class Demo
+public class Demo
 {
 	private readonly PairedNuimoManager _pairedNuimoManager = new PairedNuimoManager();
 	private INuimoController _nuimoController;
 
-	private async Task GetPairedNuimos()
+	private async void GetPairedNuimos()
 	{
 		var nuimoControllers = await _pairedNuimoManager.ListPairedNuimosAsync();
 		_nuimoController = nuimoControllers.ElementAt(0);
 	}
 
-	private async Task Connect()
+	private async void Connect()
 	{
 		var isConnected = await _nuimoController.ConnectAsync();
 	}
 
 	private void AddDelegates()
 	{
-		_nuimoController.GestureEventOccurred     += OnNuimoGestureEvent;
-		_nuimoController.FirmwareVersionRead      += OnFirmwareVersion;
-		_nuimoController.ConnectionStateChanged   += OnConnectionState;
+		_nuimoController.GestureEventOccurred += OnNuimoGestureEvent;
+		_nuimoController.FirmwareVersionRead += OnFirmwareVersion;
+		_nuimoController.ConnectionStateChanged += OnConnectionState;
 		_nuimoController.BatteryPercentageChanged += OnBatteryPercentage;
-		_nuimoController.LedMatrixDisplayed       += OnLedMatrixDisplayed;
+		_nuimoController.LedMatrixDisplayed += OnLedMatrixDisplayed;
```

Comment:
Hi @Inrego, sorry for responding so late. Can you please undo the formatting change here? The spaces are kept here intentionally to improve reading.